ser/train.py

```diff
@@ -25,7 +30,7 @@ def train(name, learning_rate, epochs, batch_size):
     learning_rate = learning_rate
 
     # load model
-    model = Net().to(device)
+    model = MnistResNet().to(device)
```

Comment:
Would be cool to configure this via the cli I think. You could store a dictionary with the different models keyed by some string, e.g.:

```
MODELS = {
    "basic": Net,
    "resnet": MnistResNet,
}
```

You can then pass the model name on the cli and access it like this:

```
klass = MODELS[model_id]
mode = klass().to(device)
```